src/molecules/teaser/index.tsx

```diff
@@ -1,53 +1,24 @@
 import * as React from "react";
 import styled from "styled-components";
-import Button from "../../atoms/button";
 import colors from "../../atoms/colors";
-import Copy, { Size } from "../../atoms/copy";
-import Headline from "../../atoms/headline";
 
 export interface TeaserProps {
-	/** @name Button label @default Label */ buttonLabel: string;
-	/** @name Copy text @default Copytext */ copyText: string;
-	/** @name Headline level @default H1 */ headlineLevel: Level;
-	/** @name Headline align @default Left */ headlineAlign?: TextAlign;
-	/** @name Headline text @default Headline */ headlineText: string;
+	/** @name Headline*/ headline: React.ReactNode;
+	/** @name Button */ button: React.ReactNode;
 	/** @ignore */ onClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
This property should be deleted as well i guess since it's not connected to the button component anymore.